README.md

```diff
@@ -0,0 +1,194 @@
+# OpenIddict, as an OpenID Connect Addict
+
+### What's Open Iddict ?
+
+OpenIddict library aims to provide a simple and easy out of the box solution to implement your OpenID Connect 
+server for ASP.NET 5.
+OpenIddict leverages the use of Identity (for user management) and Entityframework (as store provider).
+
+### What's Open ID Connect ?
+
+Open ID is a protocol that allows multiple clients to delegate the Authentication to one or multiple Authority servers.
+Facebook, Google, Twitter, are all well-known as OpenId Authority Servers.
+
+### Why should I want/need my own Open ID Connect Server?
+
+There are many scenarios on when this may be useful, 
+
+ 1. Having multiple applications with one central place for authentication.
+ 2. Having non-trusted clients (like JavaScript, Desktop or Mobile applications).
```

Comment:
Actually, the client are not necessarily non-trusted. You can have totally trusted MVC clients.
I guess you meant external clients, where you want to allow third-party developers to access your users' data?